apps/total-typescript/src/hooks/use-mux-player.tsx

```diff
@@ -46,6 +44,22 @@ type VideoProviderProps = {
   path?: string
   muxPlayerRef: any
   onEnded?: () => Promise<any>
+  videoThumbId?: string
+}
+
+const useNextLesson = (
+  lesson: VideoResource,
+  module: SanityDocument,
+  section?: SanityDocument,
+) => {
+  const router = useRouter()
+  const {data: nextExercise} = trpc.lessons.getNextLesson.useQuery({
+    type: lesson._type,
+    slug: router.query.exercise as string,
```

Comment:
the route contains data and we can use that which is super useful for determining if we want the exercise or the solution as the current lesson

this will also work if we use /ide as well to present the stackblitz or other interactive step